### PR TITLE
Adding a beta note for profiling, slack/pypi badges

### PR DESCRIPTION
**Pull request by kolanos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +135,8 @@ IOpipe's functionality can be extended through plugins. Plugins hook into the ag
 
 ### Profiler Plugin
 
+**Note:** This feature is still in beta. Want to try it out? Find us on [Slack](https://iopipe-community.slack.com/).
```
Comment (pselle) [inline README.md]: This slack link should be to iopipe.now.sh, for the signup to slack option

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # IOpipe Analytics & Distributed Tracing Agent for Python
 
+[![PyPI version](https://badge.fury.io/py/iopipe.svg)](https://badge.fury.io/py/iopipe)
```
Comment (pselle) [inline README.md]: Yay badges! This is great

Comment (kolanos) [reply README.md]: @pselle I assume we want both slack links to go to iopipe.now.sh?

**Comment by pselle** [comment]
Ah, if it's mirroring existing, keep it the same. Separate change then. But
I think we want it to the signup link in general.

File: README.md (hunk 2 above)
Comment (pselle) [reply README.md]: Ignore previous comment. Yes, both to now.sh.

**Pull request merged**